prime-router/docs/getting_started.md

```diff
@@ -329,3 +332,41 @@ Alternatively, inject them in your terminal with (useful for using the CLI):
 ```bash
 export $(cat ./.vault/env/.env.local | xargs)
 ```
+
+
+## TroubleShooting
+
+### prime test Utility
+
+The prime-router comes packaged with a executable that can help in finding misconfigurations and other problems with the appliciation.
+
+Use the following command to launch the tool. 
+
+```shell
+./prime-router/prime test
+```
+
+This should be used while the prime-router application is running on your system.
+
+
+
+#### Missing env var
+
+The gradle script is supposed to install a few environment variables that will be used by the prime router, however this is currently prone to failure. If a run of `prime test` mentions that any are missing try manually adding them: 
+
+```shell
+POSTGRES_PASSWORD='changeIT!'
```

Comment:
Note that only ./prime test depends on these vars.  Other features of the ./prime cli work fine.

The fix, btw, is to build an audit-trail/history API so that we can query an endpoint for data, rather than having to pull data from sql queries.

Reply:
Okay, I reworded the text to hopefully be more accurate.